libc/test/src/__support/freelist_test.cpp

```diff
@@ -8,159 +8,43 @@
 
 #include <stddef.h>
 
-#include "src/__support/CPP/array.h"
-#include "src/__support/CPP/span.h"
 #include "src/__support/freelist.h"
 #include "test/UnitTest/Test.h"
 
-using LIBC_NAMESPACE::FreeList;
-using LIBC_NAMESPACE::cpp::array;
-using LIBC_NAMESPACE::cpp::byte;
-using LIBC_NAMESPACE::cpp::span;
+namespace LIBC_NAMESPACE_DECL {
```

Comment:
In General we try to avoid putting the tests inside of the `LIBC_NAMESPACE` to ensure that we're getting our implementation of a function instead of one pulled from the system. I think it's probably fine to do `using` here since it's only testing internals, but I'd like to avoid the `namespace LIBC_NAMESPACE_DECL`.

Reply:
Done.